genet/schedule_elements.py

```diff
@@ -267,7 +323,36 @@ def __init__(self, route_short_name: str, mode: str, stops: List[Stop], trips: D
             self.await_departure = []
         else:
             self.await_departure = await_departure
-        super().__init__(stops)
+        if kwargs:
+            if 'ordered_stops' in kwargs:
+                ordered_stops = kwargs.pop('ordered_stops')
+            if '_graph' in kwargs:
+                _graph = kwargs.pop('_graph')
+            self.add_additional_attributes(kwargs)
+
+        if ordered_stops is not None:
+            if _graph is not None:
+                # check graph type and schema
+                verify_graph_schema(_graph)
+                self._graph = _graph
+            else:
+                raise RouteInitialisationError('When passing `ordered_stops` you are expected to pass `_graph` too. '
+                                               'You may prefer to pass a list of Stop objects to `stops` instead')
+            # check all stops in _graph
+            if not self._stop_ids_in_graph(ordered_stops):
+                raise RouteInitialisationError('Some stop IDs passed in `ordered_stops` are missing from the _graph '
+                                               'object passed')
+            self.ordered_stops = ordered_stops
```

Comment:
this is all to do with initiating a route from a shared schedule graph

Reply:
but notice they are hidden in kwargs because it's not something a user should concern themselves with